metriq-api/api-routes.js

```diff
@@ -39,6 +39,8 @@ router.route('/password')
   .post(accountController.password)
 router.route('/user/password')
   .post(accountController.update_password)
+router.route('/user/update')
+  .post(userController.update)
```

Comment:
@vprusso Look at line 46 below, in your branch: the route is already `POST /api/user`, whether for insertion or update. Point your route requests there.